gazetteer-style.hpp

```diff
@@ -12,6 +12,7 @@ class gazetteer_style_t
 {
     using flag_t = uint16_t;
     using ptag_t = std::pair<char const *, char const *>;
+    using ptag_str_t = std::pair<const std::string, const std::string>;
```

Comment:
'char const *' will do for the first parameter. It is only used with constant strings.